facilitator/src/manifest.rs

```diff
@@ -61,10 +61,12 @@ pub struct SpecificManifest {
 
 /// Obtains a manifest file from the provided URL
 fn fetch_manifest(manifest_url: &str) -> Result<Response> {
-    if !manifest_url.starts_with("https://") {
-        return Err(anyhow!("Manifest must be fetched over HTTPS"));
-    }
-    let response = ureq::get(manifest_url)
+    let url_with_scheme = if !manifest_url.starts_with("https://") {
```

Comment:
Style-wise I tend to prefer to accept just one type of input - i.e. require the `https://` prefix - and fix whatever is providing the wrong sort of input. Is there a particular issue this is solving?

Reply:
No, ultimately these values come from the `ingestors`, `peer_share_processor_manifest_base_url` and `portal_server_manifest_base_url` variables in `<env>.tfvars`, and we can make sure that they're `https://` there. What I want is to make sure that we're fetching these manifests over HTTPS.